db/seeds.rb

```diff
@@ -10,3 +10,4 @@
 topics_list.each do |name|
   Topic.create(name: name)
 end
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```

Comment:
Small comment here, `password_confirmation` is not needed, and remember to always leave an empty line at the end of the file (I think you can configure this on your editor to be automatic) :)